engine/src/main/java/org/terasology/rendering/dag/nodes/OpaqueObjectsNode.java

```diff
@@ -90,7 +98,7 @@ public void disableWireframe() {
     public void process() {
         PerformanceMonitor.startActivity("rendering/opaqueObjects");
 
-        READ_ONLY_GBUFFER.bind(); // TODO: remove when we can bind this via a StateChange
+        sceneOpaqueFbo.bind(); // TODO: remove when we can bind this via a StateChange
```

Comment:
Same as previous comments, this variable won't be updated if resizing or swapping occurs.

BindFBO should work though. I suspect that's something you could rely on and the problem is elsewhere.